### PR TITLE
Append letters game board

### PR DESCRIPTION
**Pull request by kayleenovak**
@JoelSmith123, any suggestions on refactoring the repetitive code within the conditionals? I plan on breaking out the if statement that changes the inner text of the correct div.

### CODE REVIEW

**Comment by kayleenovak** [comment]
@thatPamIAm @brittanystoroz pull request for Wheel of Fortune

File: index.html [hunk 1]
```diff
@@ -47,35 +47,57 @@ <h2 class='player3-score'>player 3 score</h2>
...
+      <div class='puzzle-grid-square tile-32'></div>
+      <div class='puzzle-grid-square tile-33'></div>
+      <div class='puzzle-grid-square tile-34'></div>
+      <div class='puzzle-grid-square tile-35'></div>
+      <div class='puzzle-grid-square tile-36'></div>
+      <div class='puzzle-grid-square tile-37'></div>
+      <div class='puzzle-grid-square tile-38'></div>
+      <div class='puzzle-grid-square tile-39'></div>
+      <div class='puzzle-grid-square tile-40'></div>
+      <div class='puzzle-grid-square tile-41'></div>
```
Comment (thatPamIAm) [inline index.html]: There are a lot of tiles to account for in your HTML. Have you thought about possibly dynamically generating these with JS? This would eliminate the need to hook into the class names to set the letters.

File: lib/Game.js [hunk 2]
```diff
@@ -33,6 +33,25 @@ class Game {
     domUpdates.updateRoundCategory(newRoundPuzzle.currentPuzzle.category)
   }
 
+  checkLetters(letter) {
+    console.log(letter);
+    // split correct answer into an array of letters, all lowercase
```
Comment (thatPamIAm) [inline lib/Game.js]: Don't commit console logs.

File: lib/Game.js [hunk 3]
```diff
@@ -33,6 +33,25 @@ class Game {
...
+    // compares user inputed letter to currentPuzzle correct letters
+    // if the user guessed letter is in the currentPuzzle correct letters
+    if (answerLettersArray.includes(letter)) {
+      console.log(letter)
+      domUpdates.appendGuessToDom(letter, correctAnswer)
+      return true
+    } else {
+      // run error message
+      return false
+    }
```
Comment (thatPamIAm) [inline lib/Game.js]: The leftover console logs and comments make the block of code pretty hard to parse.

File: domUpdates.js [hunk 4]
```diff
@@ -38,6 +38,55 @@ const domUpdates = {
...
+           if(letter === currentGuess) {
+            var thisDiv = document.querySelector(`.tile-${column}`);
+            thisDiv.innerText = currentGuess;
+          }
+          column += 1
+        })
+       }
+     })
+   }
+
```
Comment (thatPamIAm) [inline domUpdates.js]: I think a good way to refactor this long method is to make sure that methods that are kept in your `domUpdates` file are only manipulating the DOM. `appendGuessToDom` should only be doing what its name state - appending the guess to the DOM. I'm a bit confused as to why this method seems to be responsible for checking/evaluating the guess against the answer as well as updating the DOM. Anything that has to do with the logic of your App (like comparing a guess to an answer) should be handled within the appropriate class.

**Comment by thatPamIAm** [comment]
@JoelSmith123 @kayleenovak Make sure that you are keeping your DOM manipulation separate from your class logic that handles data. I think once you do this, you'll find it easier to refactor the longer conditionals you have.
